infra/managed-cluster/aks-cluster.bicep

```diff
@@ -18,16 +21,16 @@ param deploymentDate string = utcNow('yyyyMMdd-HHmmss')
 
 var kubernetesVersion = '1.26.6'
```

Comment:
I'd want this var in a var file I think!

Reply:
Moved version to parameters